docs/design/ingest_gwo.md

```diff
@@ -0,0 +1,20 @@
+# Adding Ingest GWO functionality
```

Comment:
I know its still in draft, but could you spell out GWO somewhere

Reply:
GNU World Order

Reply:
Geodetic Wildebeest Organizer